Makefile

```diff
@@ -50,10 +53,21 @@ $(PROJECT_DEBS): reset-deb-dir
 $(GOBIN):
 	mkdir -p $(GOBIN)
 
-test:
+test-go:
 	@go version
-	go test ./test/go ./copydb/test ./sharding/test -p 1 -v -count 1
-	bundle install && bundle exec rake test DEBUG=1 TESTOPTS="-v"
+	ulimit -n 1024
+
+	@if [ ! -f $(GOBIN)/gotestsum ]; then \
+		curl -sL $(GOTESTSUM_URL) | tar -xz -C $(GOBIN) gotestsum; \
+	fi
```

Comment:
Installing a random binary to people's GOPATH/bin as a part of the side effects of the test seems a bit sketchy.. especially since this is a public project.

Maybe we can temporarily install this to `./bin/`? At some point in the near-ish future I can convert this project using `nix-shell` (and dev.yml with nix).

Reply:
You are right, want to avoid adding the dependencies to the project. Changing it to `./bin` as you suggest.